libcodechecker/server/api/product_server.py

```diff
@@ -201,17 +211,12 @@ def getCurrentProduct(self):
 
             server_product = self.__server.get_product(prod.endpoint)
             if not server_product:
-                # TODO: Like above, better support this.
-                LOG.error("Product '{0}' was found in the configuration "
-                          "database, but no database connection is "
-                          "present. Was the configuration database "
-                          "connected to multiple servers?"
-                          .format(prod.endpoint))
-                LOG.info("Please restart the server to make this "
-                         "product available.")
-                raise shared.ttypes.RequestFailed(
-                    shared.ttypes.ErrorCode.DATABASE,
-                    "Product exists, but was not connected to this server.")
+                LOG.info("Product '{0}' was found in the configuration "
```

Comment:
Refactor this piece of code to a separate function and use it at the `getProducts` function and here because the big part of these functions are the same.

https://github.com/whisperity/CodeChecker/blob/e1a4cb65341b58324d7b8445770b650abaebc198/libcodechecker/server/api/product_server.py#L147-L181

https://github.com/whisperity/CodeChecker/blob/e1a4cb65341b58324d7b8445770b650abaebc198/libcodechecker/server/api/product_server.py#L212-L239